src/components/model-item/model-item.style.js

```diff
@@ -85,9 +86,10 @@ export const useStyles = makeStyles((theme) => ({
     fontSize: '14px',
     fontWeight: '600',
     display: 'flex',
+    alignItems: 'center',
     '& svg': {
       marginLeft: '5px',
-      fontSize: '1.2em'
+      fontSize: '1.7em'
```

Comment:
please use similar dimensions
check your other files for this issue

Reply:
i got it